opentelemetry-exporters-newrelic-auto/src/main/java/com/newrelic/telemetry/opentelemetry/export/auto/NewRelicSpanExporterFactory.java

```diff
@@ -0,0 +1,32 @@
+package com.newrelic.telemetry.opentelemetry.export.auto;
+
+import com.newrelic.telemetry.Attributes;
+import com.newrelic.telemetry.opentelemetry.export.NewRelicSpanExporter;
+import io.opentelemetry.sdk.contrib.auto.config.Config;
+import io.opentelemetry.sdk.contrib.auto.config.SpanExporterFactory;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+
+public class NewRelicSpanExporterFactory implements SpanExporterFactory {
+
+  private static final String NEWRELIC_API_KEY = "newrelic.api.key";
+  private static final String NEWRELIC_SERVICE_NAME = "newrelic.service.name";
```

Comment:
I *think* that this should just be `service.name`. This is the canonical name for the service, which should be being provided via the Resource in the SDK. However, the standard way for how to get this value into the Resource is not currently well defined.

Reply:
Hmm. Looking at the jaeger exporter factory, your choice is ok for now. This is definitely something we'll need to keep an eye on as the SDK specification evolves.

Reply:
I named it as `newrelic.service.name` based on jaeger exporter factory. Let me know if I need to update it